src/components/content/conditions/conditions.stories.tsx

```diff
@@ -19,6 +20,20 @@ type Props = ConditionsProps;
 export default {
   component: Conditions,
   tags: ["docsPage"],
+  argTypes: {
+    className: {
+      options: ["Blank", "Fixed Width"],
+      control: "select",
+      mapping: {
+        Blank: "",
+        "Fixed Width": "ctw-m-auto ctw-max-w-[600px]",
+      },
+    },
+  },
+  args: {
+    className: "Empty",
```

Comment:
guessing at this, should "Empty" map to a key of `argTypes.className.mapping`
```suggestion
    className: "Blank",
```
if not just ignore

Reply:
No that's correct, fixed!